apps/studio/src/pages/home/wallet/Wallet.tsx

```diff
@@ -89,76 +99,120 @@ const Wallet: FunctionComponent = () => {
-      <Box mx={ [null, null, 1.5] }>
-        <Box
-          sx={ {
-            alignItems: "center",
-            display: "flex",
-            justifyContent: "space-between",
-            mb: 5,
-          } }
-        >
-          <Typography fontWeight={ 800 } variant="h3">
-            WALLET
-          </Typography>
-          <DisconnectWalletButton />
+  if (isClientConfigLoading) return;
+
+  if (!isClaimWalletRoyaltiesEnabled) {
+    return (
+      <Container maxWidth={ false }>
+        <Box mx={ [null, null, 3] }>
+          <Box
+            sx={ {
+              alignItems: "center",
+              display: "flex",
+              justifyContent: "space-between",
+              mb: 5,
+            } }
+          >
+            <Typography fontWeight={ 800 } variant="h3">
+              WALLET
+            </Typography>
+            { wallet ? (
+              <DisconnectWalletButton />
+            ) : (
+              <Button
+                gradient="crypto"
+                sx={ { mb: 5, mr: [0, 4.75] } }
+                width="compact"
+                onClick={ () => dispatch(setIsConnectWalletModalOpen(true)) }
+              >
+                Connect Wallet
+              </Button>
+            ) }
+          </Box>
+          <Box mt={ 5 } pb={ 5 }>
+            <Typography variant="body1">
+              Wallet Portfolio and Transactions Coming Soon
+            </Typography>
+          </Box>
         </Box>
-
-        { songs.length === 0 && !isLoading ? null : unclaimedRoyalties ? (
-          <UnclaimedRoyalties unclaimedRoyalties={ unclaimedRoyalties } />
-        ) : (
-          <NoPendingEarnings />
-        ) }
-
-        <Box mt={ 5 } pb={ 5 }>
-          <Box borderBottom={ 1 } borderColor={ theme.colors.grey400 }>
-            <Tabs
-              aria-label="Wallet details"
-              sx={ {
-                ".Mui-selected": {
-                  background: theme.gradients[colorMap[tab]],
-                  backgroundClip: "text",
-                  color: theme.colors[colorMap[tab]],
-                  textFillColor: "transparent",
-                },
-                ".MuiButtonBase-root.MuiTab-root": {
-                  minWidth: "auto",
-                  ...theme.typography.subtitle2,
-                  color: theme.colors.grey400,
-                  fontWeight: 500,
-                },
-                ".MuiTabs-flexContainer": {
-                  gap: 4,
-                  justifyContent: ["center", "center", "normal"],
-                },
-                ".MuiTabs-indicator": {
-                  background: theme.gradients[colorMap[tab]],
-                },
-              } }
-              value={ tab }
-              onChange={ handleChange }
-            >
-              <Tab aria-controls="tabpanel-0" id="tab-0" label="PORTFOLIO" />
-              <Tab aria-controls="tabpanel-1" id="tab-1" label="TRANSACTIONS" />
-            </Tabs>
+      </Container>
+    );
+  } else {
```

Comment:
This is where the new Wallet Enhancement features come into play, this is the same code that was already part of this enhancement.